docs/src/dedicated-gen-3/options.md

```diff
@@ -1,25 +1,36 @@
 ---
 title: "Optional features"
 weight: 4
+description: Add optional features to your Dedicated project.
 ---
 
+As a Dedicated user, you can add the following optional features to your project.
+
 ## Multiple applications
```

Comment:
Multi-app is not really an option on DG3. It's built-in for Grid and DG3 and part of the product.